pageserver/src/tenant/timeline/eviction_task.rs

```diff
@@ -58,14 +58,16 @@ impl Timeline {
             false,
             async move {
                 self_clone.eviction_task(task_mgr::shutdown_token()).await;
-                info!("eviction task finishing");
                 Ok(())
             },
         );
     }
 
     #[instrument(skip_all, fields(tenant_id = %self.tenant_id, timeline_id = %self.timeline_id))]
     async fn eviction_task(self: Arc<Self>, cancel: CancellationToken) {
+        scopeguard::defer! {
+            info!("eviction task finishing");
+        }
```

Comment:
Similarly here, the other shutdown messages must be removed if this is added.

Reply:
Again, what other shutdown messages? Please link to them, I think this defer is an exact swap for the info in the calling function.

Reply:
L89R91: `info!("shutting down");`
L104R106: `info!("shutting down");`